src/main/webapp/components/results/results.js

```diff
@@ -313,6 +330,14 @@ const Results = props => {
   )
 }
 
+const query = gql`
```

Comment:
Since this file is kind of large, can we give a more descriptive name to this query?

Reply:
renamed to attributeAliasQuery